staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -77,6 +77,8 @@ type Scale struct {
 }
 
 // +genclient
+// +genclient:method=GetScale,verb=get,subresource=scale,result=Scale
```

Comment:
This looks very wrong.  Having scale defined locally like this means that either someone tweaked the hpa controller (very naughty) or it doesn't work with hpa, right?

Reply:
@deads2k yeah, I wondered why apps.Scale exists... Thought there is just one Scale that should be used everywhere.

Reply:
@DirectXMan12 @deads2k I check the `podautoscaler/horizontal.go` and what it does is `scale = Get() -> ext.Scale`  and then `Update(kind, scale)`...

It seems like the Scale struct in this group was copied by mistake and it is not used anywhere? Also I guess we should have just one Scale (looks like autoscaling group would be a good destination for this). WDYT?

Reply:
@deads2k i don't think this is a blocker for this PR, once we start plumbing the code to use this new client methods, the first thing we have to do is reconcile the `Scale` usage in kube and regenerate the client to pickup whatever destination we pick for it. Agree?

Reply:
> @deads2k i don't think this is a blocker for this PR, once we start plumbing the code to use this new client methods, the first thing we have to do is reconcile the Scale usage in kube and regenerate the client to pickup whatever destination we pick for it. Agree?

It's not a blocker.  I don't want you to change types here.  I think someone was confused.